Testing/integration/allow_sysex.scpt

```diff
@@ -0,0 +1,30 @@
+on run argv
```

Comment:
nit: comment this file. 

Which host should we be running this on etc. Explain that we need this to get rid of the stale pref.

Could we use automator at all for any of this?